scarb/src/core/manifest/toml_manifest.rs

```diff
@@ -198,6 +200,13 @@ pub struct TomlPackage {
     /// **UNSTABLE** This package does not depend on Cairo's `core`.
     pub no_core: Option<bool>,
     pub cairo_version: Option<MaybeWorkspaceField<VersionReq>>,
+    pub experimental: Option<MaybeWorkspaceField<ScarbExperimentalFeaturesConfig>>,
```

Comment:
Suggestion: what do you think about the following alternative interface:

```toml
[package]
name = "foobar"
experiments = ["negative_impls"]
```

1. It's much more concise.
2. I do not expect to have to pass any parameters here, actually. Like, we are turning on/off flags here only.

Reply:
Changed that to match your suggestion, but I still don't like the fact that with each new compiler flag we'll need to add support for it instead of doing it automatically - or maybe I'm missing something.

Reply:
We can write unit tests that will ensure everything is covered -- this should reduce the pain of maintaining this.

Reply:
Added info to the #1040 issue about the test and raising a warning